hardhat.config.ts

```diff
@@ -124,7 +124,7 @@ const config: HardhatUserConfig = {
                 settings: {
                     optimizer: {
                         enabled: true,
-                        runs: 200
+                        runs: 180
```

Comment:
contract size issue? this might have some unexpected implications

Reply:
Yeah, it’s over the contract size without reducing the optimizer runs

Reply:
We can remove the method that allows whitelisting multiple addresses at once

Reply:
removed